scripts/qa_crowbarsetup.sh

```diff
@@ -4460,6 +4460,10 @@ function onadmin_ping_running_instances
 
 function onadmin_testpostupgrade
 {
+    if ! grep non_disruptive /var/lib/crowbar/upgrade/*-progress.yml ; then
```

Comment:
I would recommend to use `grep -q ...`

Btw. is complain that what you want? This would fail the job.
Maybe `return 1` (meaning false) would be better in this case?

Reply:
I think failing the job is right in this situation. Otherwise it will fail later as it will try to do "non-disruptive" action to a disruptive job